
<!DOCTYPE html>
<html class="">
<head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>Monocle GenePattern Module</title>
  <link href="./assets/docstyle.css" media="all" rel="stylesheet">
  <style type="text/css"></style>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta content="authenticity_token" name="csrf-param">
</head>
<body >

<script>
</script>
	
	<div class="gp-content-header fluid">
		<div class="container-fluid">
			<div class="row">
				<div class="col-md-12">
			        <h4><img src="./assets/gplogo.png"/>Module Documentation</h4>
                                <h1><big>Monocle</big></h1>
				</div>
			</div>
		</div>
	</div>
	<div class="container-fluid">
		<div class="row">
			<div class="col-md-12">
				
<br/>
<dl class="dl-horizontal">
	<dt>LSID</dt>
	<dd>urn:lsid:broad.mit.edu:cancer.software.genepattern.module.analysis:00412</dd>
	<dt>Author(s)</dt>
	<dd>Karchin Lab, Johns Hopkins University, wrapped as a module by Ted Liefeld, Mesirov Lab, UCSD School of Medicine.</dd>
	<dt>Contact(s)</dt>
	<dd><p>Algorithm and scientific questions:&nbsp;&lt;support &nbsp;https://cole-trapnell-lab.github.io/monocle3/ &gt;</p>
	<p>Module wrapping issues:&nbsp; Ted Liefeld &nbsp;&lt; jliefeld at cloud dot ucsd dot edu&gt;</p>
	</dd>
</dl>

<h2>Introduction</h2>
<p>Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
</p>

<h2>References</h2>
<p><b>Documentation:</b> <a href="https://cole-trapnell-lab.github.io/monocle3/">Trapnell lab github documentation</a></p>

<h2>Tool Description</h2>

<h3>Input File</h3>

<ul style="list-style-type:none;">
	<li>tab delimited counts matrix</li>
	<li>&nbsp;</li>
	next release of the tool will provide a selection for older genome version (e.g. GRCh37/hg19)</li>
</ul>

<h3>Output Files</h3>

<ul style="list-style-type:none;">
	<li><b>&lt;XXXXX&gt;.csv.zip</b><br/>
	   The zip file contains .
        </li>
</ul>


<h3>Example Data</h3>

<p>	Small sample input files: <a href="">SAMPLE INPUT</a> </p> 

<h3>Requirements</h3>

<p>GenePattern 3.9.11 or later (dockerized).</p>


<h3>Parameters</h3>

<table class="table table-striped">
	
	<tbody>
		<tr><td colspan=2 ><h4>Inputs and Outputs</h4></td></tr>
		<tr>
			<th>Name</th>
			<th>Description</th>
		</tr>
		<tr>
			<td>data file*</td>
			<td>Input file in one of the supported OpenCRAVAT input file formats like VCF.</td>
		</tr>
        <tr>
            <td>XXXX</td>
            <td>XXXX</td> 
        </tr>
     </tbody>
</table>
</body></html>

